java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/OFConnectionURI.java

```diff
@@ -0,0 +1,54 @@
+package org.projectfloodlight.openflow.types;
+
+import io.netty.buffer.ByteBuf;
+import org.projectfloodlight.openflow.annotations.Immutable;
+import org.projectfloodlight.openflow.exceptions.OFParseError;
+
+import com.google.common.hash.PrimitiveSink;
+import com.google.common.primitives.UnsignedInts;
+
+@Immutable
+public class OFConnectionURI implements OFValueType<OFConnectionURI>  {
+	
+    private final int connectionURI;
```

Comment:
clarification question, because this is an integer -- this identifies an index into a connection table? I.e., is `0` the first connection, `1` the second etc?

If so, I would maybe prefer calling this `OfConnectionIndex`, because I would expect an object called `-Uri` to, well, hold an URI...